pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1386,6 +1381,16 @@ func (cf *changeFrontier) Start(ctx context.Context) {
 		return
 	}
 
+	if err = checkpoint.Restore(cf.frontier, cf.spec.SpanLevelCheckpoint); err != nil {
```

Comment:
nit: use `err :=`

you should reduce scope of variables whenever possible https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/181371303/Go+Golang+coding+guidelines#Reduce-Scope-of-Variables